Code/src/main/java/nl/utwente/group10/ui/components/Block.java

```diff
@@ -66,6 +66,8 @@ public void setSelected(boolean selectedState) {
 		isSelected = selectedState;
 	}
 	
+	public abstract String getName();
```

Comment:
Javadoc ontbreekt: wat voor naam is dit, waar gebruik je het voor, waarom moet elk Block het per se overriden? Wat is er mis met `toString`?